src/layout-grid/styled-components.js

```diff
@@ -70,19 +72,21 @@ export const StyledGrid = styled<StyledGridPropsT>(
 export const StyledCell = styled<StyledCellPropsT>(
   'div',
   ({
+    $theme,
     $align = null,
     $gridColumns = null,
     $gridGaps = 0,
     $gridGutters = null,
+    $gridUnit = $theme.grid.unit,
     $order = null,
     $skip = [0, 0, 0],
     $span = [1, 1, 1],
-    $theme,
   }) => {
     const mediaQueries = getMediaQueries($theme.breakpoints);
     const gridColumns = $gridColumns || $theme.grid.columns;
     const gridGaps = $gridGaps || $theme.grid.gaps;
     const gridGutters = $gridGutters || $theme.grid.gutters;
```

Comment:
If we move the defaults into the params destructure, maybe we should do away with the renaming and just use the `$gridUnit`, `$gridGaps`, etc props directly, the same as the other props (`$span`, `$order`, etc). Whichever you choose let's just keep it consistent between StyledCell and StyledGrid.

Reply:
I believe moving the defaults into the params destructure makes code more readable. I have made changes accordingly. :)